<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="2" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-26T19:26:34 IST" name="Default suite" finished-at="2022-11-26T19:27:55 IST" duration-ms="81123">
    <groups>
    </groups>
    <test started-at="2022-11-26T19:26:34 IST" name="Default test" finished-at="2022-11-26T19:27:55 IST" duration-ms="81123">
      <class name="Project_AMS.MyExceptionHistory">
        <test-method is-config="true" signature="initilize()[pri:0, instance:Project_AMS.MyExceptionHistory@4310d43]" started-at="2022-11-26T19:26:34 IST" name="initilize" finished-at="2022-11-26T19:27:19 IST" duration-ms="44914" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initilize -->
        <test-method signature="validateException()[pri:0, instance:Project_AMS.MyExceptionHistory@4310d43]" started-at="2022-11-26T19:27:19 IST" name="validateException" finished-at="2022-11-26T19:27:34 IST" duration-ms="14211" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateException -->
        <test-method signature="validateExceptionHistoryStatus()[pri:1, instance:Project_AMS.MyExceptionHistory@4310d43]" started-at="2022-11-26T19:27:34 IST" name="validateExceptionHistoryStatus" finished-at="2022-11-26T19:27:55 IST" duration-ms="21908" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateExceptionHistoryStatus -->
      </class> <!-- Project_AMS.MyExceptionHistory -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
